content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -58,6 +58,8 @@ In version 2.6.0 and later, running `prisma introspect` with an existing schema
 - `@default(uuid())` (`uuid()` is a Prisma-level function)
 - Custom `@relation` names
 
+> **Note**: Only relations between models on the database level will be picked up. This means that there **must be a foreign key set**.
```

Comment:
This is not really in the correct location. This section is about Re-Introspection (so introspecting when you already have a schema). 

This Note also applies to normal Introspections, so we in theory should have a list of "what" it introspects and then could mention that there. Probably should live under "The prisma introspect command", and say that tables becomes models, columns become fields, foreign keys becomes relations, indexes stay just that and so on.

cc @molebox

Reply:
Oh ok, good call! https://github.com/prisma/docs/issues/1766